docs/molgenis/dev_profiles.md

```diff
@@ -0,0 +1,70 @@
+|-----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ontologiesToFixedSchema     | Load any ontologies into a separate schema with the specified name.                                                                                                                                                                                                                                                                                                                                                                                  |
+| setViewPermission           | Apply view permissions to specified user or role to the imported schema and ontologies. Includes the schema specified by ontologiesToFixedSchema.                                                                                                                                                                                                                                                                                                    |
+| setEditPermission           | Apply editor permissions to specified user or role to the imported schema and ontologies. Includes the schema specified by ontologiesToFixedSchema.                                                                                                                                                                                                                                                                                                  |
+| firstCreateSchemasIfMissing | Before creating the schema and ontologies specified by this profile, import one more more other profiles first. Required 3 additional properties to be provided: name, profile and importDemoData. Name specified the name under which the other profile should be imported. Profile points to the profile YAML file. Lastly, importDemoData is either true or false and specified whether the demo data for this profile should be imported or not. |
+
+## Complete example
+
+This example profile contains all currently supported options:
+
+```yaml
+---
+name: SharedStaging
+description: "This schema contains communal tables that can be altered by all users."
+profileTags: SharedStaging
+demoData: _demodata/applications/datacatalogue_sharedstaging
```

Comment:
What file formats are allowed? Is it only csv file formats?

Reply:
I think it's all formats, but I'm not really sure. For me, only CSV makes sense because changes can be tracked instead of having a binary blob like XLSX.